Userland/Libraries/LibJS/CyclicModule.h

```diff
@@ -40,6 +40,7 @@ class CyclicModule : public Module {
     Vector<ModuleWithSpecifier> const& loaded_modules() const { return m_loaded_modules; }
     Vector<ModuleWithSpecifier>& loaded_modules() { return m_loaded_modules; }
 
+    virtual ~CyclicModule() override;
```

Comment:
nit: let's keep the newline before the `protected:` label
super duper nit: let's keep dtors and ctors nearer the top of the declarations